docs/02quickstart.md

```diff
@@ -1,2 +1,75 @@
 # 快速开始
 
+这一节介绍如何快速部署一个单Master RocketMQ集群，并完成简单的消息收发。
+
+**（1）下载Apache RocketMQ**
+
+RocketMQ的安装包分为两种，二进制包和源码包。点击[这里](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-source-release.zip) 下载Apache RocketMQ 4.9.4的源码包。你也可以从[这里](https://www.apache.org/dyn/closer.cgi?path=rocketmq/4.9.4/rocketmq-all-4.9.4-bin-release.zip) 下载到二进制包。二进制包是已经编译完成后可以直接运行的，源码包是需要编译后运行的，以在Linux环境下利用社区4.9.4的源码包为例，我们介绍RocketMQ安装过程
+
+解压4.9.4的源码包并编译构建二进制可执行文件
```

Comment:
Prerequisite for building and running RocketMQ should be supplemented, such as the version of maven, jdk, and so on.